README.md

```diff
@@ -132,6 +132,10 @@ element in the array is a line in the requirements file.
 ```
 _Sample `command.json`._
 
+See the [Command Schema](
+https://github.com/mesosphere/universe/blob/master/repo/meta/schema/command-schema.json) for a
```

Comment:
Could we use the relative form of the link for this?  See https://github.com/blog/1395-relative-links-in-markup-files